app/views/AccessibilityAndAppearanceView/components/ListPicker.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import { TActionSheetOptionsItem, useActionSheet } from '../../../containers/ActionSheet';
+import { CustomIcon } from '../../../containers/CustomIcon';
+import * as List from '../../../containers/List';
+import I18n from '../../../i18n';
+import { useTheme } from '../../../theme';
+import sharedStyles from '../../Styles';
+import { TAlertDisplayType } from '..';
+
+const styles = StyleSheet.create({
+	leftTitleContainer: {
+		flex: 1,
+		justifyContent: 'center',
+		alignItems: 'flex-start'
+	},
+	leftTitle: {
+		...sharedStyles.textMedium,
+		fontSize: 16,
+		lineHeight: 24
+	},
+	rightContainer: {
+		flex: 1
+	},
+	rightTitle: {
+		...sharedStyles.textRegular,
+		fontSize: 16,
+		lineHeight: 24
+	},
+	rightTitleContainer: {
+		flex: 1,
+		justifyContent: 'center',
+		alignItems: 'flex-end'
+	}
+});
+
+type TOPTIONS = { label: string; value: TAlertDisplayType; description: string | null }[];
+
+const OPTIONS: TOPTIONS = [
+	{
+		label: I18n.t('A11y_appearance_toasts'),
+		value: 'TOAST',
+		description: I18n.t('A11y_appearance_toast_dismissed_automatically')
```

Comment:
We talked about it recently.
This is not going to work correctly, since the i18n is going to be applied only once.
On my tests, everything is `en` except this, that's `pt-BR`.